velodyne_pointcloud/include/velodyne_pointcloud/transform.h

```diff
@@ -61,7 +61,7 @@ namespace velodyne_pointcloud
 class Transform final : public rclcpp::Node
 {
 public:
-  Transform();
+  explicit Transform(const rclcpp::NodeOptions& options);
   ~Transform()
```

Comment:
Can you please remove the explicit constructor? With this here, the compiler won't generate copy or move operations and it isn't doing anything.

Reply:
I don't care hugely about this, so I'll go either way here.  But in my (limited) testing here, having an empty destructor doesn't seem to disable the copy constructor.  I'm less certain about the move constructor (it's a little harder to write a straightforward example that shows it).

But actually, I'd argue that we explicitly don't *want* a copy or move constructor.  There's enough resources in here that I'm not at all certain we'd do the right thing on copy.  From that perspective, it might be better to introduce a `NO_COPY_NO_MOVE_NO_ASSIGN` macro (like [here](https://github.com/ros-drivers/phidgets_drivers/blob/10c1ca6ab712398c6c398b85a175ae20ae3e442e/phidgets_api/include/phidgets_api/phidget22.hpp#L38)), and explicitly not allow those operations.

What do you think?

Reply:
I'm OK with removing the copy and move operators. Here is the proper way:

```
Transform(Transform && t) = delete;
Transform &operator=(Transform && t) = delete;
Transform(const Transform &) = delete;
Transform &operator=(const Transform &) = delete;
```

Reply:
Great, thanks.  Fixed in 3ec6c52